app/models/spree/order_decorator.rb

```diff
@@ -0,0 +1,15 @@
+module SpreeGateway
+  module OrderDecorator
```

Comment:
@dimon-k let's rename this please to `ApplePayOrderDecorator` to be more specific

Reply:
Fixing